Gemfile

```diff
@@ -26,9 +26,9 @@ gem 'git', '~> 1.2.5'
 gem 'oily_png', '~> 1.1.1'
 gem 'nokogiri', '~> 1.5.10'
 gem 'therubyracer', platforms: :ruby, require: linux_only('therubyracer')
-gem 'aweplug', github: 'awestruct/aweplug'
+gem 'aweplug', github: 'Dantheman720/aweplug', :ref => 'c5cf1eb4a43720ebd19967982d18625540bd3148'
```

Comment:
Use `:branch => 'github_token'` rather than the commit id. This makes it clearer to see which branch you are pointing to. 

It also makes it clearer to see that there is a mistake ;-) Yu should be pointing at the 'github_token_header' branch I believe.